src/mqtt/fbws.js

```diff
@@ -0,0 +1,93 @@
+//From mqtt.js
+'use strict'
+
+var websocket = require('websocket-stream')
+var urlModule = require('url')
+var WSS_OPTIONS = [
+  'rejectUnauthorized',
+  'ca',
+  'cert',
+  'key',
+  'pfx',
+  'passphrase'
+]
+var IS_BROWSER = process.title === 'browser'
```

Comment:
Are we actually doing something with the browser or process.title?

Reply:
Actually no, this is copied from MQTT.js itself.
I can look into unused things like this and remove it.